Assets/MixedRealityToolkit-Examples/Demos/UX/Layout3D/Scripts/AutoCycletLayout3DSize.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Layout3D
+{
+    public class AutoCycletLayout3DSize : MonoBehaviour
```

Comment:
Cyclet?

Reply:
Thanks for catching that, I'll update shortly.